src/sinks/datadog/metrics/encoder.rs

```diff
@@ -867,7 +1028,7 @@ mod tests {
         let expected_interval = interval_ms / 1000;
 
         // Encode the metric and make sure we did the rate conversion correctly.
-        let result = generate_series_metrics(&rate_counter, &None, log_schema());
+        let result = generate_series_metrics(&rate_counter, &None, log_schema(), 14);
```

Comment:
Nit: we could pull up these constants to actual constants. I realize they aren't used, typically, in more than one place but it could make it easier to reason about them.